src/Tests/dotnet.Tests/GivenThatDotNetRunsCommands.cs

```diff
@@ -40,21 +40,6 @@ public void UnresolvedPlatformReferencesFailAsExpected()
                      .And.HaveStdErrContaining(string.Format(LocalizableStrings.NoExecutableFoundMatchingCommand, "dotnet-crash"));
         }
 
-        [Theory]
-        [InlineData("")]
-        [InlineData(null)]
-        public void GivenAMissingHomeVariableItPrintsErrorMessage(string value)
```

Comment:
If it doesn't fail anymore, then please test the new behavior.

Reply:
Will do. Thank you.